company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md

```diff
@@ -32,7 +36,11 @@ _Example 2_: a staff member named John Smith is located in Spain and signed a co
 * The bank account is located in Spain
 * The bank account is in the name of Better Coding, LLC
 
-If you are a staff member located outside of the United States and Canada, your expense reimbursements will be made through Bill.com. As noted above, Mattermost requires banking information from a primary banking institution to be able to make reimbursements through Bill.com. You will receive your expense reimbursement no later than 15th and end of each month. 
+If you provide information for your banking institution, your invoice and/or expense reimbursements will be made through Bill.com. As noted above, Mattermost requires banking information from a primary banking institution to be able to make reimbursements through Bill.com. You will receive your expense reimbursement no later than 15th and end of each month. 
+
+TransferWise
+
+If you provide information for your TransferWise account, your invoice and/or expense reimbursements will be made through TransferWise. As noted above, you will need to be a staff member located outside of the United States, Canada, United Kingdom and Germany to be able to choose this option. If your consulting agreement is other than your name such as an LLC or LLP, than we request you set up a business account for contracting entity in TransferWise under the same legal name. Mattermost primarily makes payments in USD, unless otherwise stated on your agreement signed with Mattermost. You will receive your expense reimbursement no later than 15th and end of each month.
```

Comment:
```suggestion
If you provide information for your TransferWise account, your invoice and/or expense reimbursements will be made through TransferWise. As noted above, you will need to be a staff member located outside of the United States, Canada, United Kingdom and Germany to be able to choose this option. If your consulting agreement is other than your name such as an LLC or LLP, than we request you set up a business account for contracting entity in TransferWise under the same legal name. Mattermost primarily makes payments in USD, unless otherwise stated on your agreement signed with Mattermost. You will receive your expense reimbursement no later than between the 15th and end of each month.
```

Reply:
```suggestion
If you provide information for your TransferWise account, your invoice and/or expense reimbursements will be made through TransferWise. As noted above, you will need to be a staff member located outside of the United States, Canada, United Kingdom and Germany to be able to choose this option. If your consulting agreement is other than your name such as an LLC or LLP, than we request you set up a business account for contracting entity in TransferWise under the same legal name. Mattermost primarily makes payments in USD, unless otherwise stated on your agreement signed with Mattermost. You will receive your expense reimbursement no later than 15th and end of each month. Approved expense reports are reimbursed either by the 15th or end of each month. 
```